spikingjelly/activation_based/op_counter/neuromc/core.py

```
@@ -142,723 +210,1208 @@ def _resolve_loss_fn(loss_fn: Callable | None):
-            move_bits_by_stage_op.setdefault("unlabeled", {})
-            for op_name, edge_info in unlabeled_move_op.items():
-                move_bits_by_stage_op["unlabeled"].setdefault(op_name, {})
-                _add_nested(move_bits_by_stage_op["unlabeled"][op_name], edge_info)
-
-        energy_compute_pj = (
-            primitive_totals["mul"] * self.op_cost["mul"]
-            + primitive_totals["add"] * self.op_cost["add"]
-            + primitive_totals["cmp"] * self.op_cost["cmp"]
-            + primitive_totals["sqrt"] * self.op_cost["sqrt"]
-            + primitive_totals["mux"] * self.op_cost["mux"]
+        for _, (count_key, bits_per_elem, reg_name, sram_name) in variables.items():
```

Ruff (0.15.12)

[warning] Loop control variable `reg_name` not used within loop body

Rename unused `reg_name` to `_reg_name`

(B007)